sevntu-checks/src/main/java/com/github/sevntu/checkstyle/checks/coding/Jsr305AnnotationsCheck.java

```diff
@@ -0,0 +1,1096 @@
+        /** ReturnValuesAreNonnullByDefault. */
+        RETURN_VALUES_ARE_NONNULL_BY_DEFAULT("ReturnValuesAreNonnullByDefault",
+                "edu.umd.cs.findbugs.annotations");
+
+        /** The annotation's name. */
+        private final String annotationName;
+
+        /** The annotation's fully qualified class name. */
+        private final String fullyQualifiedClassName;
+
+        /**
+         * Constructor.
+         * @param annotationName
+         *        the annotation's name
+         * @param packageName
+         *        the package name
+         */
+        NullnessAnnotation(final String annotationName, final String packageName) {
+            this.annotationName = annotationName;
+            this.fullyQualifiedClassName = packageName + "." + annotationName;
+        }
+
+    }
+
+    /** The map of annotations against their respective names. */
+    private static final Map<String, NullnessAnnotation> STRING2ANNOTATION =
+            createString2AnnotationMap();
+
+    /** The modifiers of interest. */
+    private static final int[] DEFAULT_MODIFIERS = {
+        TokenTypes.PARAMETER_DEF,
+        TokenTypes.METHOD_DEF,
+        TokenTypes.PACKAGE_DEF,
+        TokenTypes.CTOR_DEF,
+        TokenTypes.CLASS_DEF,
+        TokenTypes.INTERFACE_DEF,
+        TokenTypes.ENUM_DEF,
+    };
+
+    /** Parameter: packages to check. */
+    private String[] packages = new String[0];
+    /** Parameter: packages to exclude from checking. */
+    private String[] excludePackages = new String[0];
+    /** Parameter: overriding return value annotations allowed. */
+    private boolean allowOverridingReturnValue;
+    /** Parameter: overriding parameter annotations allowed. */
+    private boolean allowOverridingParameter;
+
+    /** State, is a package excluded. */
+    private boolean packageExcluded;
+
+    @Override
+    public final int[] getDefaultTokens() {
+        return getAcceptableTokens();
+    }
+
+    @Override
+    public final int[] getRequiredTokens() {
+        return new int[0];
+    }
+
+    @Override
+    public final int[] getAcceptableTokens() {
+        return DEFAULT_MODIFIERS.clone();
+    }
+
+    @Override
+    public final void visitToken(final DetailAST ast) {
+        if (ast.getType() == TokenTypes.PACKAGE_DEF) {
+            final DetailAST nameAST = ast.getLastChild().getPreviousSibling();
+            packageExcluded = isPackageExcluded(FullIdent.createFullIdent(nameAST));
+        }
+        else if (!packageExcluded) {
+            final AbstractJsr305Handler handler = handleDefinition(ast);
+            if (handler != null) {
+                handler.check();
+            }
+        }
+    }
+
+    /**
+     * Sets the included packages parameter.
+     * @param packageNames
+     *        the package names, comma separated
+     */
+    public void setPackages(final String... packageNames) {
```

Comment:
please move all parameter setters to be above `Override` methods, and they should be grouped together.
and please rename "parameter" to "property", to keep terminology consistent - https://checkstyle.org/config.html#Properties

Reply:
Done.

Reply:
Please recheck what you pushed to remote, I still see this as not done